### PR TITLE
Allow solution/projects to load in either VS 2015 or VS 2017.

### PR DESCRIPTION
**Pull request by mcnameej**
Solution will load into either VS version, and automatically use the toolset from that version.

### CODE REVIEW

File: samples/samples.props [hunk 1]
```diff
@@ -1,20 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ImportGroup Label="PropertySheets" />
-  <PropertyGroup Label="UserMacros">
-    <EDK_PATH>$(SolutionDir)..\edk2</EDK_PATH>
-  </PropertyGroup>
   <PropertyGroup>
     <GenerateManifest>
```
Comment (ionescu007) [inline samples/samples.props]: Want to fold this up like you did in the other file?

Comment (mcnameej) [inline samples/samples.props]: Done.

Comment (ionescu007) [inline samples/samples.props]: Hmm, the diff is still showing <GenerateManifest> having a second </GenerateManifest> underneath. There are two commits but they only touch the readme file and another props file. Did you forget to push?

Comment (mcnameej) [inline samples/samples.props]: Updated.

**Comment by ionescu007** [comment]
Thanks @mcnameej. By the way, Norton Utilities and the related suite tools on Windows later on are the reason why I got into Windows Internals. It's an honor to have you submit a PR :)

**Comment by mcnameej** [comment]
Damn, I feel old now.  😈

simpsons-meme-it-will-happen-to-you

**Pull request merged**